src/modules/75-cf/pages/feature-flags/components/StaleFlagActions/__tests__/StaleFlagActions.test.tsx

```diff
@@ -42,4 +43,48 @@ describe('StaleFlagActions', () => {
     expect(screen.queryByRole('button', { name: 'cf.staleFlagAction.readyForCleanup' })).not.toBeInTheDocument()
     expect(screen.queryByRole('button', { name: 'cf.staleFlagAction.learnMore' })).not.toBeInTheDocument()
   })
+
+  test('it should render the info drawer when info button is clicked', async () => {
+    renderComponent()
+
+    userEvent.click(screen.getByRole('button', { name: 'cf.staleFlagAction.learnMore' }))
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'close' })).toBeVisible()
+      expect(screen.getByText('cf.staleFlagAction.flagLifecycleExplained')).toBeVisible()
+      expect(screen.getByText('cf.staleFlagAction.flagLifecycleDesc')).toBeVisible()
+      expect(screen.getByRole('button', { name: 'common.gotIt' })).toBeVisible()
+    })
+  })
+
+  test('it should close the info drawer when the x button is clicked', async () => {
+    renderComponent()
+
+    userEvent.click(screen.getByRole('button', { name: 'cf.staleFlagAction.learnMore' }))
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'close' })).toBeVisible()
+    })
+
+    userEvent.click(screen.getByRole('button', { name: 'close' }))
+
+    await waitFor(() => {
+      expect(screen.queryByRole('button', { name: 'close' })).not.toBeInTheDocument()
+    })
+  })
+
+  test('it should close the info drawer when the got it button is clicked', async () => {
+    renderComponent()
+
+    userEvent.click(screen.getByRole('button', { name: 'cf.staleFlagAction.learnMore' }))
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'common.gotIt' })).toBeVisible()
+    })
+
+    userEvent.click(screen.getByRole('button', { name: 'common.gotIt' }))
+
+    await waitFor(() => {
+      expect(screen.queryByRole('button', { name: 'common.gotIt' })).not.toBeInTheDocument()
+    })
```

Comment:
Also not an issue, but _typically_ with the later versions of Testing Library, if you await the action, you don't need to `waitFor` the response. So for things like this, it gets a wee bit simpler. 100% not an issue, just an FYI.
```suggestion
    await userEvent.click(screen.getByRole('button', { name: 'common.gotIt' }))

    expect(screen.queryByRole('button', { name: 'common.gotIt' })).not.toBeInTheDocument()
```